addons/mail/static/tests/message/message_reply.test.js

```diff
@@ -120,3 +121,27 @@ test("click on message in reply highlights original message", async () => {
     );
     await contains(".o-mail-Message.o-highlighted:contains('This message has been removed')");
 });
+
+test("reply action functionality in Chatter", async () => {
+    const pyEnv = await startServer();
+    const partnerBId = pyEnv["res.partner"].create({ name: "Partner B" });
+    pyEnv["mail.message"].create({
+        body: "Test message from B",
+        subtype_id: pyEnv["mail.message.subtype"].search([
+            ["subtype_xmlid", "=", "mail.mt_note"],
+        ])[0],
+        model: "res.partner",
+        res_id: serverState.partnerId,
+        author_id: partnerBId,
+    });
+    await start();
+    await openFormView("res.partner", serverState.partnerId);
+    await click(".o-mail-Message button[title='Reply']");
+    await contains("button.active", { text: "Log note" });
+    await contains(".o-mail-Composer.o-focused .o-mail-Composer-input", { value: "@Partner B " });
+    await click(".o-mail-Composer button", { text: "Log" });
+    await contains(".o-mail-Message a.o_mail_redirect", { text: "@Partner B" });
+    await click(".o-mail-Message:contains('@Partner B') button[title='Expand']");
+    await contains(".o-dropdown-item", { text: "Forward" });
+    await contains(".o-dropdown-item", { count: 0, text: "Reply" });
```

Comment:
For the last assertion `count: 0` of `Reply` to be valuable, we need an equivalent selector that is positive.
One solution is to add another message in setup of test for which the `Reply` action is in the dropdown, to compare the one with `Reply` with the one without the `Reply`.